chia/types/mempool_item.py

```diff
@@ -21,8 +23,9 @@ class MempoolItem(Streamable):
     spend_bundle_name: bytes32
     additions: List[Coin]
     removals: List[Coin]
+    height_added_to_mempool: uint32
 
-    def __lt__(self, other):
+    def __lt__(self, other: _T_MempoolItem) -> bool:
```

Comment:
cant we use MempoolItem here ?

Reply:
Correct. This might have been before the global
`from __future__ import annotations` or some similar mypy / Python version issue.

Thanks!